### ISSUE TITLE
react-s-alert notifications are not read by narrator

### ISSUE THREAD

**Issue by lavanyakuda** [juliancwirko/react-s-alert-demo#3]
React-s-alert notifications are not read by the narrator.
Screen reader is unable to read the react-s-alert notifications pop out.
We use "react-s-alert": "1.4.1" version in our application
